spec/rails_helper.rb

```diff
@@ -53,3 +56,9 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
+
+VCR.configure do |config|
+  config.cassette_library_dir = 'spec/fixtures/vcr_cassettes'
+  config.hook_into :webmock
+  config.configure_rspec_metadata!
+end
```

Comment:
We'll need to cover the API key here, all of the recordings will have it exposed currently!